cli/go.mod

```diff
@@ -15,7 +15,7 @@ require (
 	github.com/google/uuid v1.1.1
 	github.com/googleapis/gnostic v0.1.0 // indirect
 	github.com/hashicorp/go-version v1.2.0
-	github.com/keptn/go-utils v0.8.0-alpha.0.20210108094001-51257d664f81
+	github.com/keptn/go-utils v0.8.0-alpha.0.20210113100130-2b6cb3d71512
```

Comment:
Let's wait until PR https://github.com/keptn/go-utils/pull/228 is merged, and then we'll update cli/go.mod here.